src/contest_effect.c

```diff
@@ -129,7 +129,7 @@ static void ContestEffect_UserLessEasilyStartled(void)
     SetContestantEffectStringID(shared192D0.contestant,CONTEST_STRING_STOPPED_CARING);
 }
 
-// Slightly startles the POK�MON in front.
+// Slightly startles the POKéMON in front.
```

Comment:
I don't understand what the problem is here. Using a UTF-8 string analyzer, both `POKéMON` displayed here are using U+00E9 for the `e`.